packages/amplify-environment-parameters/package.json

```diff
@@ -24,7 +24,7 @@
     "clean": "rimraf lib tsconfig.tsbuildinfo node_modules src/schemas",
     "test": "jest --logHeapUsage",
     "extract-api": "ts-node ../../scripts/extract-api.ts",
-    "generate-schemas": "mkdirp lib/schemas src/schemas && ts-json-schema-generator --path 'src/backend-parameters.d.ts' --type 'BackendParameters' --no-type-check --out lib/schemas/BackendParameters.schema.json && cp lib/schemas/BackendParameters.schema.json src/schemas/BackendParameters.schema.json"
+    "generate-schemas": "mkdirp lib/schemas src/schemas && ts-json-schema-generator --path src/backend-parameters.d.ts --type BackendParameters --no-type-check --out lib/schemas/BackendParameters.schema.json && copyfiles --flat lib/schemas/BackendParameters.schema.json src/schemas"
```

Comment:
why did we leave out BackendParameters.schema.json?

Reply:
no, that's intentional. It will maintain the source filename.